frontend/public/components/routes.tsx

```diff
@@ -116,13 +116,13 @@ export const RouteStatus: React.SFC<RouteStatusProps> = ({obj: route}) => {
       </React.Fragment>;
     case 'Accepted':
       return <React.Fragment>
-        <div className="fa fa-check text-success co-m-route-status-icon" aria-hidden="true"></div>
-      Accepted
+        <div className="fa fa-check route-accepted co-m-route-status-icon" aria-hidden="true"></div>
+        <span className="route-accepted">Accepted</span>
       </React.Fragment>;
```

Comment:
Any reason not to use...? (Here and below)

```jsx
return <span className="route-accepted">
  <span className="fa fa-check co-m-route-status-icon" aria-hidden="true"></span>
  Accepted
</span>;
```

Missed this on previous review, but the icon should probably be an inline element like `<span>` or `<i>` instead of a block element  (`<div>`).

Reply:
> Any reason not to use...? (Here and below)

Not sure I understand your question, by ? if you mean inline if, it's not a "this or that" -it can be three values Pending, Accepted, Rejected.

Reply:
Any reason not to use the code I pasted with one route-accepted class on the parent span.

Reply:
> ...icon should probably be an inline element like <span> or <i> instead of a block element (<div>).
Fixed, thanks